lib/elixir_boilerplate_web/health/plug.ex

```diff
@@ -2,9 +2,14 @@ defmodule ElixirBoilerplateWeb.Health.Plug do
   use Plug.Builder
 
   def call(%{request_path: "/health"} = conn, _) do
+    version =
+      :elixir_boilerplate
+      |> Application.spec(:vsn)
+      |> String.Chars.to_string()
+
     conn
     |> put_resp_header("content-type", "application/json")
-    |> send_resp(200, Jason.encode!(%{status: "ok", version: Application.spec(:elixir_boilerplate, :vsn)}))
+    |> send_resp(200, Jason.encode!(%{status: "ok", version: version}))
```

Comment:
Your right, running `Application.spec(:elixir_boilerplate, :vsn)` in `iex` was "misleading" since it automatically converts to string, which `Jason.encode/1` does not!

Reply:
> since it automatically converts to string

It renders the list of chars as `'1.0.0'` so it can be misleading yes but `iex` does not convert output to string (like `"1.0.0"`) 😄

Reply:
This is what I meant, but I failed to explain it right 😬